runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -402,7 +402,7 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 				}
 			}
 
-			if (config.showMenuItemQuantities() && itemComposition.isStackable() && quantity > 1)
```

Comment:
Why is this removed?

Reply:
Because now entire right click menu can be disabled from being recolored, so I felt like having option for this is unnecessary as it is good default behaviour.